src/main/java/uk/gov/hmcts/reform/prl/services/dynamicmultiselectlist/DynamicMultiSelectListService.java

```diff
@@ -110,6 +115,29 @@ public Map<String, List<DynamicMultiselectListElement>> getRespondentsMultiSelec
         return respondentdetails;
     }
 
+    public void updateChildrenWithCaseCloseStatus(CaseData caseData, Element<OrderDetails> order) {
+
+        List<Element<Child>> children = caseData.getChildren();
+        String childrenFromOrder = order.getValue().getChildrenList();
+        List<String> childrenList = Stream.of(childrenFromOrder.split(","))
+            .map(String::trim)
+            .map(elem -> new String(elem))
+            .collect(Collectors.toList());
+        if (C100_CASE_TYPE.equalsIgnoreCase(caseData.getCaseTypeOfApplication())
+            && finl.equals(caseData.getSelectTypeOfOrder())
+            && Yes.equals(caseData.getServeOrderData().getDoYouWantToServeOrder())
+            && null != children) {
+            for (Element<Child> child:children) {
+                String childName = child.getValue().getFirstName() + " " + child.getValue().getLastName();
```

Comment:
line 131. write up something like this
@JsonIgnore
    public String getLabelForDynamicList() {
        return String.format(
            "%s %s",
            this.firstName,
            this.lastName
        );
    }